Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5724,6 +5724,20 @@ ScrollAnimatorEnabled:
     WebCore:
       default: false
 
+ScrollDrivenAnimationsEnabled:
+  type: bool
+  status: unstable
```

Comment:
You want this to be `testable` if you want the feature to be enabled in tests (and see WPT progressions for your changes).